test/Microsoft.Azure.ServiceBus.UnitTests/QueueClientTests.cs

```diff
@@ -174,7 +174,7 @@ public sealed class QueueClientTests : SenderReceiverClientTestBase
             var queueClient = (QueueClient)messagingFactory.CreateQueueClientFromConnectionString(TestUtility.GetEntityConnectionString(queueName), ReceiveMode.ReceiveAndDelete);
```

Comment:
(QueueClient) cast can be removed for all the tests and the object can be directly passed everywhere instead of InnerReceiver or InnerSender